notes/fomega/cek-cps-experiments/src/CekMachine.recursive.hs

```diff
@@ -20,24 +20,24 @@
 -- The CEK machine generates booleans along the way which might contain globally non-unique 'Unique's.
 -- This is not a problem as the CEK machines handles name capture by design.
 
-module Language.PlutusCore.Interpreter.CekMachine
+module PlutusCore.Interpreter.CekMachine
     ( EvaluationResult (..)
     , evaluateCek
     , runCek
     ) where
 
-import           Language.PlutusCore
-import           Language.PlutusCore.Constant
-import           Language.PlutusCore.Evaluation.MachineException (MachineError (..), MachineException (..))
-import           Language.PlutusCore.Evaluation.Result           (EvaluationResult (..))
-import           Language.PlutusCore.View
+import           PlutusCore
+import           PlutusCore.Constant
+import           PlutusCore.Evaluation.MachineException (MachineError (..), MachineException (..))
+import           PlutusCore.Evaluation.Result           (EvaluationResult (..))
+import           PlutusCore.View
 import           PlutusPrelude
 
-import qualified Data.Text                                       as T
-import qualified Language.PlutusCore.Pretty                      as PLC
+import qualified Data.Text                              as T
+import qualified PlutusCore.Pretty                      as PLC
```

Comment:
Shouldn't this be up above, with all the other `PlutusCore` things?

Reply:
I guess this stuff doesn't get formatted... I don't care too much about the stuff in `notes` tbh.